f2/src/forms/ContactInfoForm.js

```diff
@@ -1,162 +1,160 @@
 /** @jsx jsx */
 import React from 'react'
-import addrs from 'email-addresses'
 import PropTypes from 'prop-types'
 import { jsx } from '@emotion/core'
 import { Trans } from '@lingui/macro'
-import { Form } from 'react-final-form'
-import { NextAndCancelButtons } from '../components/next-and-cancel-buttons'
-import { Stack } from '@chakra-ui/core'
 import { useStateValue } from '../utils/state'
-import { ErrorSummary } from '../components/ErrorSummary'
-import { Input } from '../components/input'
-import { Field } from '../components/Field'
-import { clientFieldsAreValid } from '../utils/clientFieldsAreValid'
-import { formatPhoneNumber } from '../utils/formatPhoneNumber'
-import { formDefaults } from './defaultValues'
-import { Flex, Icon } from '@chakra-ui/core'
 import { P } from '../components/paragraph'
-import { Button } from '../components/button'
+import { Form, Container, Row } from 'react-bootstrap'
+import { Formik, FieldArray, Field } from 'formik'
+import { TextArea } from '../components/formik/textArea'
+import { SkipButton, NextCancelButtons } from '../components/formik/button'
+import { Flex, Icon } from '@chakra-ui/core'
```

Comment:
These Chakra-UI components should not be used.

Reply:
Resolved in new commit